lib/ocpp/v16/charge_point_impl.cpp

```diff
@@ -56,9 +60,15 @@ ChargePointImpl::ChargePointImpl(const std::string& config, const std::filesyste
     bool log_to_html = std::find(log_formats.begin(), log_formats.end(), "html") != log_formats.end();
     bool session_logging = std::find(log_formats.begin(), log_formats.end(), "session_logging") != log_formats.end();
 
+	#ifndef BOOSTFILESYSTEM
     this->logging = std::make_shared<ocpp::MessageLogging>(
-        this->configuration->getLogMessages(), message_log_path, DateTime().to_rfc3339(), log_to_console,
-        detailed_log_to_console, log_to_file, log_to_html, session_logging);
+    		        this->configuration->getLogMessages(), message_log_path.string(), DateTime().to_rfc3339(), log_to_console,
+    		        detailed_log_to_console, log_to_file, log_to_html, session_logging);
+	#else
+		this->logging = std::make_shared<ocpp::MessageLogging>(
+		        this->configuration->getLogMessages(), message_log_path.string(), DateTime().to_rfc3339(), log_to_console,
+		        detailed_log_to_console, log_to_file, log_to_html, session_logging);
+	#endif
```

Comment:
Don't see any difference between the two. :thinking:

Reply:
In the beyond part the snd argument, uses message_log_path.string()

Reply:
Actually, there's literally no difference between the two.